src/Files.App/Constants.cs

```diff
@@ -124,16 +124,13 @@ public static class GridViewBrowser
 			{
 				public const int GridViewIncrement = 20;
 
-				// Max achievable ctrl + scroll, not a default layout size
-				public const int GridViewSizeMax = 300;
-
 				public const int GridViewSizeLarge = 220;
 
 				public const int GridViewSizeMedium = 160;
 
 				public const int GridViewSizeSmall = 100;
 
-				public const int TilesView = 260;
+				public const int TilesView = 100;
```

Comment:
The correct size is `100`, the only reason it was set to `260` was to set the width in the of the xaml item. There is more refactoring to be done but that's out of scope for this PR.